src/pages/profile-view.page.js

```diff
@@ -157,7 +157,7 @@ export default class ProfileViewPage extends Component {
             fullWidth
             color="primary"
             variant="outlined"
-            onClick={() => this.routeChange('/map')}
+            href="/logout"
```

Comment:
since we are using react router, all actions that trigger internal routing actually need to occur within a <Link> component, even if they are caused by buttons. see header-solid line 31

Reply:
you can see how the app quickly reloads when you use an href tag, and wont when you use a button